### PR TITLE
getting-started-guides-cloudstack-pr-2017-07-27

### PR DESCRIPTION
**Pull request by angelacy**

### CODE REVIEW

File: docs/getting-started-guides/cloudstack.md [hunk 1]
```diff
@@ -4,20 +4,28 @@ assignees:
...
+<!--
 This guide uses an [Ansible playbook](https://github.com/apachecloudstack/k8s).
 This is completely automated, a single playbook deploys Kubernetes.
 
 This [Ansible](http://ansibleworks.com) playbook deploys Kubernetes on a CloudStack based Cloud using CoreOS images. The playbook, creates an ssh key pair, creates a security group and associated rules and finally starts coreOS instances configured via cloud-init.
-
-
+-->
+这份文档使用[Ansible playbook](https://github.com/apachecloudstack/k8s).
+使用一种完全自动化地，单一的脚本去部署 Kubernetes
```
Comment (markthink) [inline docs/getting-started-guides/cloudstack.md]: This [Ansible](http://ansibleworks.com) playbook deploys Kubernetes on a CloudStack based Cloud using CoreOS images. The playbook, creates an ssh key pair, creates a security group and associated rules and finally starts coreOS instances configured via cloud-init.
 
这里少一段内容未译

Comment (angelacy) [reply docs/getting-started-guides/cloudstack.md]: I have add this part, please help to review it again, thanks

Comment (angelacy) [reply docs/getting-started-guides/cloudstack.md]: Please help to review it

**Comment by angelacy** [comment]
@BruceAuyeung please help to review it, thanks

**Comment by bruceauyeung** [comment]
@angelacy  @markthink sorry for the delay. 
does this merged commit still need another review ?

**Comment by angelacy** [comment]
@markthink, both @bruceauyeung  and I do not sure if this merged commit still need another review ? or I has already merge to mast branch?

**Pull request merged**